### PR TITLE
Rename Distribution to Histogram.

### PR DESCRIPTION
**Pull request by bogdandrutu**

### CODE REVIEW

File: opencensus/proto/metrics/metrics.proto [hunk 1]
```diff
@@ -147,13 +147,10 @@ message Point {
   }
 }
 
-// Distribution contains summary statistics for a population of values. It
-// optionally contains a histogram representing the distribution of those
-// values across a set of buckets.
-message DistributionValue {
+// Histogram contains summary statistics for a population of values.
```
Comment (songy23) [inline opencensus/proto/metrics/metrics.proto]: In this case histogram buckets are required for `HistogramValue`?
What if users only want the summary stats (sum, count, mean, etc.) without a histogram?

Comment (songy23) [reply opencensus/proto/metrics/metrics.proto]: SGTM

**Comment by ramonza** [comment]
What's the motivation for this? Would be good to record it.

**Comment by bogdandrutu** [comment]
No need for this PR because we will use our own format and the current definition is compatible with Prometheus, SD, OpenMetrics etc.

**Pull request closed without merging**